src/java.base/share/native/libfdlibm/e_asin.c

```diff
@@ -99,10 +99,10 @@ qS4 =  7.70381505559019352791e-02; /* 0x3FB3B8C5, 0xB12E9282 */
                 if(huge+x>one) return x;/* return x with inexact if x!=0*/
             } else
                 t = x*x;
-                p = t*(pS0+t*(pS1+t*(pS2+t*(pS3+t*(pS4+t*pS5)))));
-                q = one+t*(qS1+t*(qS2+t*(qS3+t*qS4)));
-                w = p/q;
-                return x+x*w;
+            p = t*(pS0+t*(pS1+t*(pS2+t*(pS3+t*(pS4+t*pS5)))));
```

Comment:
should we add an opening brace after the `else` (and close it before line 102) to make it more obvious that there is no problem with the indentation?

Reply:
It seems that this code is almost a verbatim copy of the upstream code available on the indicated link.
So, while I agree that it would be better to add braces, it would make the code diverge from the upstream source, even if the change would be benign. and welcome

Reply:
OK - that's a reasonable concern indeed.